blue_hardware_interface/include/blue_hardware_interface/blue_hardware_interface.h

```diff
@@ -63,23 +65,32 @@ class BlueHW: public hardware_interface::RobotHW
   BlueHW(ros::NodeHandle &nh);
   void read();
   void write();
+  void doSwitch(const std::list<hardware_interface::ControllerInfo>& start_list,
```

Comment:
might be worth adding the `override` keyword to these three functions

Reply:
to add override would we have to match the function decelerations of `read` and `write` from the parent class?
https://github.com/ros-controls/ros_control/blob/melodic-devel/hardware_interface/include/hardware_interface/robot_hw.h

Reply:
yeah, but if we aren't matching the function signature we might be doing this inheritance thing wrong oops
seems like we can revisit in a future PR...

Reply:
ok ill open an issue for this